packages/react/src/SegmentedControl/SegmentedControl.module.css

```diff
@@ -65,6 +65,12 @@
     /* stylelint-disable-next-line primer/spacing */
     margin-right: -1px;
   }
+
+  [data-component='counter'] {
```

Comment:
Would it be possible for this to be a class name itself instead of a data attribute? If I remember right, the `data-component` pattern only shows up when targeting a child component. It seems like in this case we are rendering the `span` directly but let me know if that's not the case!